DevTools/HackStudio/main.cpp

```diff
@@ -776,12 +781,16 @@ bool make_is_available()
 {
     pid_t pid;
     const char* argv[] = { "make", "--version", nullptr };
-    if ((errno = posix_spawnp(&pid, "make", nullptr, nullptr, const_cast<char**>(argv), environ))) {
+    posix_spawn_file_actions_t action;
+    posix_spawn_file_actions_init(&action);
+    posix_spawn_file_actions_addopen(&action, STDOUT_FILENO, "/dev/null", O_RDONLY, 0);
```

Comment:
I think the reason make gives that error is you are opening `STDOUT_FILENO` for reading, rather than writing.

Reply:
Thank you, change to `O_WRONLY` fixes the issue.